frontend/sozisel-app/src/components/SessionDetails/SessionCustomization/SessionCustomization.scss

```diff
@@ -1,6 +1,6 @@
 @import "../../../styles/styles";
 
-.sessionDetailsContainer {
+.SessionDetailsContainer {
```

Comment:
Why not `SessionDetails`? Just to name some other nested class `sessionDetails`?